docs/source/getting-started/installing/linux_with_conda.rst

```diff
@@ -94,7 +94,7 @@ but this command will ensure that for everyone.
     # Note geos no longer required for cartopy >= 0.22
     # openblas / gcc required for recenter_tc / akima build.
     # git required for -C commands
-    mamba create -y -n geoips -c conda-forge python=3.10 gcc gxx openblas git
+    mamba create -y -n geoips -c conda-forge python=3.11.0 gcc gxx openblas git
```

Comment:
I think we can just say `3.11` here rather than `3.11.0`. That will allow bug fixes to still come in.